yarn-project/txe/src/index.ts

```diff
@@ -1,19 +1,15 @@
 import { SchnorrAccountContractArtifact } from '@aztec/accounts/schnorr';
-import {
-  AztecAddress,
-  type ContractInstanceWithAddress,
-  Fr,
-  type NoirCompiledContract,
-  PublicKeys,
-  deriveKeys,
-  getContractInstanceFromInstantiationParams,
-  loadContractArtifact,
-} from '@aztec/aztec.js';
+import { Fr } from '@aztec/foundation/fields';
 import { createSafeJsonRpcServer } from '@aztec/foundation/json-rpc/server';
 import type { Logger } from '@aztec/foundation/log';
 import { type ProtocolContract, protocolContractNames } from '@aztec/protocol-contracts';
 import { BundledProtocolContractsProvider } from '@aztec/protocol-contracts/providers/bundle';
```

Comment:
We could use the lazy provider here if we add `{ with: { type: 'json' } }` to the relevant `await import` line.

Reply:
These: https://github.com/AztecProtocol/aztec-packages/blob/df8d413b7053aabc9807c8835c270c1f53c45d8f/yarn-project/protocol-contracts/src/instance-registry/lazy.ts#L19

That comment shouldn't apply anymore since Firefox supports the feature as of now: https://caniuse.com/mdn-javascript_statements_import_import_attributes_type_json